### PR TITLE
Re-arranging security section

### PR DESCRIPTION
**Pull request by cloudmaniac**
## Describe the Change

This PR re-arrange the new Security section. 

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket]()

### CODE REVIEW

File: content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+|Kubernetes|Spectro Cloud Responsibility|Customer Responsibility|
+|Storage|Spectro Cloud Responsibility|Customer Responsibility|
+|Networking|Spectro Cloud Responsibility|Customer Responsibility|
+|Add-Ons|Spectro Cloud & Customer Responsibility|Customer Responsibility|
+
+We ensure our out-of-the-box core layers are secure. Customers are responsible for security if they bring their own packs. Palette provides defaults for its out-of-the-box add-on layers based on third-party best practices. Customers have the flexibility to tune the configuration to best fit their needs, making it a shared responsibility.
+
+## Cloud Infrastructure Security
+
+When deployed in a public cloud, Kubernetes nodes from tenant clusters are deployed within a logically isolated virtual network (e.g., VPC in AWS) that has private and public subnets. The control plane and worker nodes for the Kubernetes cluster are launched in a private network. All ports on the nodes are protected from external access.
```
Comment (ritawatson) [inline content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md:38]:
```suggestion
In a public cloud, Kubernetes nodes in tenant clusters are deployed within a logically isolated virtual network that has private and public subnets. The control plane and worker nodes for the Kubernetes cluster are launched in a private network. All ports on the nodes are protected from external access.
```

File: content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+- Applies the latest available security updates.
+
+
+- Hardens SSH server parameters, network parameters (sysctl), and system files by ensuring proper file permissions are set.
+
+
+- Removes legacy services and graphical user interface (GUI) packages.
+
+Palette allows you to set up OS patching policies. You can patch the base OS when you deploy the cluster. Refer to the [OS Patching](/clusters/cluster-management/os-patching) documentation to learn more.
```
Comment (ritawatson) [inline content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md:61]:
```suggestion
Palette allows you to set up OS patching policies. You can patch the base OS when you deploy the cluster. Refer to [OS Patching](/clusters/cluster-management/os-patching) to learn more.
```

File: content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## Hardened Kubernetes
+
+Spectro Cloud has a fully automated Kubernetes verification system that adds the newest patch version of Kubernetes to its public repository. All Kubernetes packs are hardened according to CIS Benchmarks.
+
+We assess major Kubernetes versions based on the extent of changes.
+
+Kubernetes run-time security support is achieved through a variety of add-on packages, such as Sysdig Falco and Twistlock.
+
+Kubernetes cluster authentication can be optionally integrated with Kubelogin using OpenID Connect (OIDC) authentication and authorization against an external Identity Provider (IDP).
```
Comment (ritawatson) [inline content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md:81]: @cloudmaniac Line 81 repeats line 89. Maybe delete line 81 and the heading in line 87 and have just one section under "Hardened Kubernetes". 

Or just delete line 81.

File: content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+You can set a schedule to start Kubernetes conformance and compliance tests using kube-bench, kube-hunter, and Sonobuoy. These tests ensure tenant clusters are secure, compliant, and up to date.
+
+<br />
+
+## Kubernetes Authentication & Authorization
+
+Kubernetes cluster authentication can be optionally integrated with Kubelogin with OIDC-based authentication/authorization against external IDP. This enables group membership-based access control on different namespaces within the tenant Kubernetes cluster. Spectro Cloud’s Terraform provider also supports automatically set role binding on namespaces with users or groups.
```
Comment (ritawatson) [inline content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md:89]:
```suggestion
Kubernetes cluster authentication can be optionally integrated with Kubelogin with OIDC-based authentication/authorization against external IDP. This enables group membership-based access control on different namespaces within the tenant Kubernetes cluster. Our Terraform provider also supports automatically set role binding on namespaces with users or groups.
```

File: content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+<br />
+
+## Compliance & Security Scans
+
+Palette provides a way to run compliance, security, conformance, and software bill of materials (SBOM) scans on tenant clusters. These scans ensure clusters adhere to specific compliance and security standards. The scans also detect potential vulnerabilities by performing penetration tests.
+
+Palette supports four types of scans. Each scan generates reports with details specific to the type of scan. You can initiate multiple scans of each type over time. In addition, Palette keeps a history of previous scans for comparison purposes.
```
Comment (ritawatson) [inline content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md:97]: What are the four types of scans? Are they compliance, security, conformance, and SBOM? 
If so, line 97 repeats. You could delete the first sentence and add "You can initiate multiple scans..." to line 95.

File: content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Overview
+
+Tenant Kubernetes clusters can be launched in the tenant’s choice of public cloud, private cloud, or bare metal environment.
+
+Palette has security controls for the operating system, containers, and Kubernetes. Tenant clusters are Kubernetes clusters deployed by Palette for customers (tenants) in their choice of public or private cloud or bare metal environment. Palette offers complete flexibility and control for designing these tenant clusters through a construct called [Cluster Profiles](/cluster-profiles).
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md:18]: Maybe you can put the following removed sentence in the Cloud Infrastructure Security section to introduce the sections that follow:

"Palette has security controls for the operating system, containers, and Kubernetes."

File: content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md
Comment (ritawatson) [inline content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md:1]: @cloudmaniac I suggest making the **Tenant Cluster Security** section the main source of information about hardened OS, containers, and Kubernetes. The same-named sections in the **Platform Security** doc should be replaced with high-level information about security for OS, containers, and Kubernetes and link to corresponding subsections in this doc.

**Review by ritawatson — CHANGES_REQUESTED** [review]
Good restructure. Offering further suggestions.

File: content/docs/09.5-security/10-product-architecture.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+# Overview
+
+Outside of our security principles and secure development lifecycle, Spectro Cloud also provides a cohesive security architecture for Palette.
+
+## Secure Product Architecture
+
+Palette uses a micro services-based architecture, and we take steps to ensure each service is secured. Product functionality is broken down logically into isolated services within containers. Containers are deployed in a Kubernetes cluster, called a management cluster, that Palette hosts and manages in SaaS mode or that users can host and manage in a self-hosted environment.
+
+Palette supports three different architecture models: multi-tenant SaaS, dedicated SaaS, and self-hosted, which includes support for air gaped environment. These flexible deployment models allow us to adapt to existing requirements in terms of separating responsibilities and network restrictions.
```
[the remark this answers is not in the document]
Comment (cloudmaniac) [reply content/docs/09.5-security/10-product-architecture.md:22]: Requirements in terms of connectivity, or responsabilities.

File: content/docs/09.5-security/11-product-architecture/01-platform-security.md
[the remark this answers is not in the document]
Comment (cloudmaniac) [reply content/docs/09.5-security/11-product-architecture/01-platform-security.md:1]: It's not a repeat: one is for the tenant clusters, and one is for Palette.

File: content/docs/09.5-security/11-product-architecture/02-data-encryption.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+<br />
+
+- **Database Communication**: The database connection from Palette internal services active in the management cluster to MongoDB is protected by TLS with Authentication enabled.
+
+<br />
+
+- **Message Bus**: A NATS message bus is used for asynchronous communication between Palette management clusters and tenant clusters. NATS messages are exchanged using TLS protocol, and each tenant cluster uses dedicated credentials to connect to the message bus. Authentication and authorization policies are enforced in the NATS deployment to ensure message and data isolation across tenants.
+
+
+## Secure Communications
```
[the remark this answers is not in the document]
Comment (cloudmaniac) [reply content/docs/09.5-security/11-product-architecture/02-data-encryption.md:66]: OK

File: content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md
Comment (cloudmaniac) [reply content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md:1]: See my other comments. Those are describing the same thing but in different contexts.

File: content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md (hunk 3 above)
Comment (cloudmaniac) [reply content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md:81]: OK, done

**Review by cloudmaniac — COMMENTED** [review]
Addressed feedback

**Review by ritawatson — APPROVED** [review]
LGTM

**Pull request merged**